.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+      - renovate/**
```

Comment:
I claim this is wrong ;-)

Reply:
Or perhaps, if it's intended, I'd suggest '**renovate**' and then document that people can test renovate stuff by using a branch vaguely named renovate -- there's no particular explanation anywhere in this PR as to why `renovate` is special as part of a branch name.

Reply:
We are using https://github.com/renovatebot to keep dependencies up to date and it creates branches that are named to match this pattern. This ensures that these branches trigger this workflow, even in the cases we have configured renovate to not open pull requests (although we still have a few kinks to work out around those cases). This pattern isn't intended to enable any special behavior for outside contributors.

Reply:
Ah, that isn't particularly obvious (there aren't enough hints anywhere). Might I suggest a `# renovate/* branches are generated by @renovate-bot `?

Reply:
great idea! Let's do that! Pull request welcome ;)